doc/getting_started/index.rst

```diff
@@ -471,71 +471,142 @@ Install a Toolchain
 A toolchain provides a compiler, assembler, linker, and other programs required
 to build Zephyr applications.
 
+The Zephyr Software Development Kit (SDK) contains toolchains for each of
+Zephyr's supported architectures. It also includes additional host tools, such
+as custom QEMU and OpenOCD.
+
 .. tabs::
 
    .. group-tab:: Ubuntu
 
-      The Zephyr Software Development Kit (SDK) contains toolchains for each of
-      Zephyr's supported architectures. It also includes additional host tools,
-      such as custom QEMU binaries and a host compiler.
-
-      |p|
-
-      #. Download the `latest SDK installer
+      #. Download and verify the `latest Zephyr SDK bundle
          <https://github.com/zephyrproject-rtos/sdk-ng/releases>`_:
 
          .. code-block:: bash
 
             cd ~
-            wget https://github.com/zephyrproject-rtos/sdk-ng/releases/download/v0.13.2/zephyr-sdk-0.13.2-linux-x86_64-setup.run
+            wget https://github.com/zephyrproject-rtos/sdk-ng/releases/download/v0.14.0/zephyr-sdk-0.14.0_linux-x86_64.tar.gz
+            wget -O - https://github.com/zephyrproject-rtos/sdk-ng/releases/download/v0.14.0/sha256.sum | shasum --check --ignore-missing
 
-      #. Run the installer, installing the SDK in :file:`~/zephyr-sdk-0.13.2`:
+      #. Extract the Zephyr SDK bundle archive:
 
          .. code-block:: bash
 
-            chmod +x zephyr-sdk-0.13.2-linux-x86_64-setup.run
-            ./zephyr-sdk-0.13.2-linux-x86_64-setup.run -- -d ~/zephyr-sdk-0.13.2
+            tar xvf zephyr-sdk-0.14.0_linux-x86_64.tar.gz
 
          .. note::
-            It is recommended to install the Zephyr SDK at one of the following locations:
+            It is recommended to extract the Zephyr SDK bundle at one of the following locations:
```

Comment:
why did you reduce the list from the original one? eg. `~/.local/` is not there any longer?

Reply:
Applies to macOS too

Reply:
It is a standard practice: https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html

Reply:
It is questionable whether that loosely defined one page doc from the XDG qualifies as a "standard practice" and not just "XDG practice," but I will add these back since we would not want to delay getting this PR merged any longer.

Reply:
As for `/usr`, I will remove it since it is outright wrong. User-installed software should not go under `/usr`; they need to go under `/usr/local`.

Reply:
added `$HOME/.local` paths back in.